src/quo2/components/avatars/group_avatar/style.cljs

```diff
@@ -9,6 +9,8 @@
    :justify-content  :center
    :border-radius    (/ container-size 2)
    :overflow         :hidden
-   :background-color (colors/theme-colors (colors/custom-color customization-color 50)
-                                          (colors/custom-color customization-color 60)
-                                          theme)})
+   :background-color (if (keyword? customization-color)
```

Comment:
@ibrkhalil I see this is in draft but I will leave some early feedback. 
It is better to leave this handling of colors to the colors resolver using `custom-color` method. 
There are many parts of the chat will be affected by this and a generic solution will be much easier to maintain and eventually update once the correct approach for colors is added.
components should not care whether it is user color, wallet account color, chat color or community color, it should all be the same mechanism

Reply:
Oh, the way I meant would mean that no change is needed to the group avatar component. All of the changes happen within the custom-color method 👍